README.md

```diff
@@ -41,15 +41,15 @@ some of these filesystems may share *parts* of their storage (i.e. some
 files may be different, but not all).
 
 Composefs ships with a mount helper that allows you to easily mount
-images by pass the image filename and the base directory for
+images by passing the image filename and the base directory for
 the content files like this:
 
 ```
 # mount -t composefs /path/to/image  -o basedir=/path/to/content /mnt
 ```
 
 By storing the files content-addressed (e.g. using the hash of the content to name
-the file) shared files need only be stored once, yet can appear in
+the file), shared files only need to be stored once, yet can appear in
```

Comment:
I believe the previous phrasing is also valid FWIW, but I'm fine with the change.